tests/unit/dojson/test_dojson_jobs.py

```diff
@@ -230,39 +230,121 @@ def test_regions_from_043__a_splits_on_commas():
 
 
 def test_experiments_from_693__e():
+
     snippet = (
-        '<datafield tag="693" ind1=" " ind2=" ">'
-        '  <subfield code="e">CERN-LHC-ATLAS</subfield>'
-        '</datafield>'
-    )  # record/1471772
```

Comment:
If you go to old record [in the legacy](http://inspirehep.net/record/1471772/export/xme) now it contains also `0` subfield:

`<datafield tag="693" ind1=" " ind2=" ">`
`<subfield code="e">CERN-LHC-ATLAS</subfield>`
`<subfield code="0">1108541</subfield>`
`</datafield>`